interface-definitions/interfaces-nat64.xml.in

```diff
@@ -0,0 +1,148 @@
+              <leafNode name="ipv4-pool">
+                <properties>
+                  <help>IPv4 dynamic pool</help>
+                  <valueHelp>
+                    <format>ipv4net</format>
+                    <description>IPv6 host will be assigned an IPv4 address from this prefix</description>
+                  </valueHelp>
+                  <constraint>
+                    <validator name="ipv4-prefix" />
+                  </constraint>
+                </properties>
+              </leafNode>
+            </children>
+          </node>
+          <node name="host-mapping">
+            <properties>
+              <help>Stateless host mappings</help>
+            </properties>
+            <children>
+              <tagNode name="ipv6">
+                <properties>
+                  <help>IPv6 address</help>
+                  <valueHelp>
+                    <format>ipv6</format>
+                    <description>IPv6 address</description>
+                  </valueHelp>
+                  <constraint>
+                    <validator name="ipv6-address"/>
+                  </constraint>
+                </properties>
+                <children>
+                  <leafNode name="description">
+                    <properties>
+                      <help>Description</help>
+                    </properties>
+                  </leafNode>
+                  <leafNode name="disable">
+                    <properties>
+                      <help>Disable this NAT64 mapping</help>
+                      <valueless/>
+                    </properties>
+                  </leafNode>
+                  <leafNode name="ipv4">
+                    <properties>
+                      <help>IPv4 address</help>
+                      <valueHelp>
+                        <format>ipv4</format>
+                        <description>IPv4 address</description>
+                      </valueHelp>
+                      <constraint>
+                        <validator name="ipv4-address"/>
+                      </constraint>
+                    </properties>
+                  </leafNode>
+                </children>
+              </tagNode>
+            </children>
+          </node>
+          <node name="ip">
+            <properties>
+              <help>IPv4 routing parameters</help>
+            </properties>
+            <children>
+              #include <include/interface/adjust-mss.xml.i>
+              #include <include/interface/source-validation.xml.i>
+            </children>
+          </node>
+          <node name="ipv6">
+            <properties>
+              <help>IPv6 routing parameters</help>
+            </properties>
+            <children>
+              #include <include/interface/adjust-mss.xml.i>
+            </children>
+          </node>
+          #include <include/interface/mtu-68-16000.xml.i>
+          #include <include/interface/mirror.xml.i>
+          #include <include/interface/netns.xml.i>
```

Comment:
Netns exist only as PoC and only with dummy interfaces
We don't use it anywhere, so you shouldn't use it anywhere as we didn't find a proper solution to integrate it.

Reply:
Removed.